array/removeAt.ts

```diff
@@ -0,0 +1,5 @@
+export default (index: number) => ([...xs]: any[]) => {
```

Comment:
Please spread within the body to keep the interface as `(index: number) => (xs: any[]): any[] => { ... }